packages/eas-build-job/src/customJob.ts

```diff
@@ -0,0 +1,135 @@
+import { z } from 'zod';
+
+const CommonCustomJobStepZ = z.object({
```

Comment:
What do you think about stripping "custom" and "job" from the names here so we focus on these being "steps"?